jerry-core/ecma/base/ecma-gc.c

```diff
@@ -242,16 +242,6 @@ ecma_gc_mark_property (ecma_property_t *property_p) /**< property */
 
       switch (ECMA_PROPERTY_GET_INTERNAL_PROPERTY_TYPE (property_p))
       {
-        case ECMA_INTERNAL_PROPERTY_NUMBER_INDEXED_ARRAY_VALUES: /* a collection of ecma values */
-        case ECMA_INTERNAL_PROPERTY_STRING_INDEXED_ARRAY_VALUES: /* a collection of ecma values */
-        {
-          JERRY_UNIMPLEMENTED ("Indexed array storage is not implemented yet.");
-        }
-
-        case ECMA_INTERNAL_PROPERTY_PROTOTYPE: /* the property's value is located in ecma_object_t
-                                                * (see above in the routine) */
-        case ECMA_INTERNAL_PROPERTY_EXTENSIBLE: /* the property's value is located in ecma_object_t
-                                                 * (see above in the routine) */
         case ECMA_INTERNAL_PROPERTY__COUNT: /* not a real internal property type,
```

Comment:
This case has a very strange placement (to me, at least). Usually, such an "impossible" case comes last in a switch. Anyone else for or against moving this to the end?